test/e2e/utils.go

```diff
@@ -733,6 +736,18 @@ func waitForElasticSearch() {
 }
 
 func getJaegerSelfProvSimpleProd(instanceName, namespace string, nodeCount int32) *v1.Jaeger {
+	var esResourcesRequest corev1.ResourceList
+	if skipESExternal {
+		esResourcesRequest = corev1.ResourceList{
+			corev1.ResourceMemory: resource.MustParse(requestMemory),
+			corev1.ResourceCPU:    resource.MustParse(requestCPU),
```

Comment:
Same as the previous comment: why do we need a CPU resource here but not in the `else` branch?